packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -292,24 +303,39 @@ contract DeployImplementations is Script {
         });
     }
 
+    // Deploy and initialize the OPStackManager contract that is fronted by a proxy.
```

Comment:
nit, "proxied" is usually the term
```suggestion
    // Deploy and initialize a proxied OPStackManager.
```

Reply:
Updated in: https://github.com/ethereum-optimism/optimism/pull/11875/commits/a77e799e4e37389132993e8883cd14678ff78d23#diff-9076b9402a282fc4bfeb1038536a8aec45ae7603f734c3f23fea747caf85f53fR303